main.py

```diff
@@ -1,15 +1,17 @@
 from modules import Clients, ServerWithGCI, GCI
 import simpy
 
-SIM_DURATION = 10
+SIM_DURATION = 12
 env = simpy.Environment()
 
-gci = GCI(env)
-server = ServerWithGCI(env, gci)
+server = ServerWithGCI(env)
 requests = list()
 clients = Clients(env, server, requests)
 
 env.run(until=SIM_DURATION)
 
+print("heap %.10f" % server.heap.level)
 print("remaining request in queue: %i" % len(server.queue.items))
-print("Processed value %.3f" % requests[0]._processed_time)
\ No newline at end of file
+print("processed requests %.3f" % len(requests))
+print("GC exe %.i" % server.gc.times_performed)
+print("GCI exe %.i" % server.gci.times_performed)
```

Comment:
Could you please also print the amount of time GCing?

Reply:
Yes! I'm thinking on do it adding the amount of time on an attribute in GCI model, to read it after the simulation.

Reply:
Done, @danielfireman could you review?

Reply:
Done! @danielfireman and @dfquaresma could you review?

Reply:
@paulofelipefeitosa is ok! I have added separate counts to know the amount of time GCing with and without the influence of GCI.